### PR TITLE
Build new /security/docker-images page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new `/security/docker-images` page 
- Add contact us and thank you pages for Docker Images
- Add to secondary nav and remove 'Overview' 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/docker-images
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1zZibtjU141e4mBxo_0PtfYxJ6e5fyaaAdFZ6v945XAM/edit#)

## Issue / Card

Fixes [#3492](https://github.com/canonical-web-and-design/web-squad/issues/3492)

### CODE REVIEW

File: templates/security/docker-images.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}{% endblock %}
```
Comment (caldav) [inline templates/security/docker-images.html:3]:
```suggestion
{% block title %}LTS Docker Images{% endblock %}
```

**Comment by valentincanonical** [comment]
Great job, few points: 
- For the "Critical CVE fixes in 24 hours" can we have some big "24h" iconography instead of repeating the blocks image? 
- Page title is "| Ubuntu", I don't know if that's because I'm on the demo URL?
- Could we have "FROM lts/redis" in mono font in the "Our Commitment" section? 
- I added comments on the Google docs concerning the FAQ section, for:
- "Are these Official Images on Docker Hub?", an updated text is available
- "Is the LTS Docker Image Portfolio a free or a commercial offering?", an updated text is available
- "Where are the images?" I'm still waiting on Docker to have the final URLs

Also, I didn't want to try it, so here's a question:
- Is the contact form directed to my email? (or some mailing list that I'm the only one to be a part of)

**Comment by valentincanonical** [comment]
as per Lech's comments: 
- replace `FROM lts/redis` with `FROM lts/nginx` in the "Our Commitment" section (request for a mono font still stands)
- drop the "Overview" link in the submenu to make it less busy

**Comment by valentincanonical** [comment]
latest tweaks on my side (also reflected on the Google Doc): 
- Read the announcement > https://ubuntu.com/blog/canonical-publishes-lts-docker-image-portfolio-on-docker-hub 
- "Where are the images?" FAQ section updated on the Google Doc to reflect the blocked situation with Docker Hub

And, finally, as soon as I get the new form from Lucia, I will let you know.

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**